uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/main/java/org/uberfire/ext/metadata/backend/infinispan/provider/InfinispanContext.java

```diff
@@ -204,10 +209,15 @@ private void addProtobufClass(SerializationContext serializationContext,
     public RemoteCache<String, KObject> getCache(String index) {
         String i = AttributesUtil.toProtobufFormat(index).toLowerCase();
         if (!this.getIndices().contains(i)) {
-            cacheManager
-                    .administration()
-                    .createCache(i,
-                                 this.infinispanConfiguration.getIndexedConfiguration(i));
+            try {
+                cacheManager
+                        .administration()
+                        .createCache(i,
+                                     this.infinispanConfiguration.getIndexedConfiguration(i));
+            } catch (CacheConfigurationException ex) {
```

Comment:
There are tons of other places in infinispan code, from which CacheConfigurationException is thrown.
Do we want to swallow all of them universally and just log the warning?
See e.g. [this](https://github.com/infinispan/infinispan/blob/ef488126e9a8199243c1cc451f6e54182c03d190/core/src/main/java/org/infinispan/configuration/parsing/XmlConfigHelper.java#L82). If there is a possibility that this exception is thrown for some Other reason (than that it already exists), the exception won't be propagated. But at least it will be logged in server.log, so perhaps that's enough.. Wdyt?